docs/baseplate/ratelimit.rst

```diff
@@ -0,0 +1,22 @@
+``baseplate.ratelimit``
+=======================
+
+.. automodule:: baseplate.ratelimit
+
+Assuming your project uses MemcacheContextFactory to attach a pymemcache client
+to the context object (or the request object for HTTP projects), you can
```

Comment:
Should this description also mention Redis  since you can use that here as well?

Reply:
will update